ui/src/app/controllers/components/ControllerHeaderForms/AddController/AddController.tsx

```diff
@@ -0,0 +1,47 @@
+import { Button, Col, Row } from "@canonical/react-components";
+import { useSelector } from "react-redux";
+
+import type { ClearHeaderContent } from "app/base/types";
+import configSelectors from "app/store/config/selectors";
+
+type Props = {
+  clearHeaderContent: ClearHeaderContent;
+};
+
+export const AddController = ({ clearHeaderContent }: Props): JSX.Element => {
+  const maasUrl = useSelector(configSelectors.maasUrl);
+  const rpcSharedSecret = useSelector(configSelectors.rpcSharedSecret);
+
+  return (
+    <>
+      <pre>
+        <code data-testid="instructions">{`# To add a new rack controller, SSH into the rack controller.
```

Comment:
Is there a particular reason for wrapping the entire content along with instructions in a code block?

It makes it more difficult to read and has an awkward horizontal scroll when the content doesn't fit in (e.g. larger font or smaller screens):

![image](https://user-images.githubusercontent.com/7452681/148983934-43965b7a-2e53-4552-a4ba-d7987fc5ef0d.png)

I know you just copied it over from the legacy version, but what about we try to improve the UX as we go?

Unless there's a good reason for it I'd suggest to move everything out of the code formatted block except the commands themselves.

Reply:
Thanks, good suggestion. I've updated it so it looks like this now:

<img width="1307" alt="Screen Shot 2022-01-12 at 9 46 08 am" src="https://user-images.githubusercontent.com/361637/149033452-6f6c9a11-1115-4241-b8dc-2a76bc2f4ddf.png">

Reply:
Awesome, looks much better.